cmd/server/assets/users/index.html

```diff
@@ -49,45 +61,105 @@
       </div>
 
       {{if $memberships}}
-        <table class="table table-bordered table-striped table-fixed table-inner-border-only border-top mb-0">
-          <thead>
-            <tr>
-              <th scope="col">Name</th>
-              <th scope="col" width="300">Email</th>
-              {{if $canWrite}}
+        <form method="POST" id="users-form">
```

Comment:
https://github.com/google/exposure-notifications-verification-server/pull/1525/files?w=1 is a better diff